### ISSUE TITLE
in CNN, track lr scheduler steps separately from epochs 

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#992]
track learning rate scheduler steps separately from epochs

say we reload a model trained for 5 epochs and want to continue training, but we specify a new lr schedule. 
we don't want to restart epochs (we have records of loss, metrics from previous epochs)
but lr scheduler needs a reference point for how many steps since initial_lr

we can just add CNN.lr_scheduler_step=0 in init and increment at same time as CNN.current_epoch

to load model then get a fresh lr schedule:
```python
m =load_model(path)
m.opt_net = None
m.scheduler = None
m.lr_scheduler_step = 0
```
